rts/at/service/service1/pom.xml

```diff
@@ -105,7 +93,8 @@
                     <workingDirectory>${project.build.directory}/exec-working-directory</workingDirectory>
                     <arguments>
                         <!--argument>-Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=8788</argument-->
-                        <argument>coordinator=http://localhost:8080/rest-at-coordinator/tx/transaction-manager</argument>
+                        <argument>coordinator=http://localhost:8080/rest-at-coordinator/tx/transaction-manager
```

Comment:
@mmusgrov could you please revert this new line?

Reply:
@marcosgopen Will this be your only comment on the PR if so I'll push a fix otherwise I'll hold back until you've finished your review. Note that this PR is stalling my other one so the sooner it gets merged the better.